glam/api/migrations/0022_sample_counts_tables.py

```diff
@@ -0,0 +1,85 @@
+# Generated by Django 3.1.13 on 2022-03-04 01:21
+
+from django.db import migrations, models
+
+from glam.api import constants
+
+sql = []
+for channel in constants.CHANNEL_NAMES.values():
+    sql.extend(
+        [
+            f"DROP MATERIALIZED VIEW view_glam_desktop_{channel}_aggregation;",
+            f"CREATE MATERIALIZED VIEW view_glam_desktop_{channel}_aggregation AS SELECT * FROM glam_desktop_{channel}_aggregation;",
+            f"CREATE UNIQUE INDEX ON view_glam_desktop_{channel}_aggregation (id);",
+            f"CREATE INDEX ON view_glam_desktop_{channel}_aggregation (version);",
+            f"CREATE INDEX ON view_glam_desktop_{channel}_aggregation USING HASH (metric);",
+            f"CREATE INDEX ON view_glam_desktop_{channel}_aggregation USING HASH (os);",
```

Comment:
I hope this won't extend our ETL time by a lot. IIRC this step usually takes the longest right?

Reply:
Yes Linh. That's what is concerning about this release.